README.md

```diff
@@ -12,90 +12,59 @@ If you're only interested in running the plugin in Sibelius, this is all you nee
 
 ## Developing and Contributing
 
-These are the instructions for installing the development dependencies and developing the plugin.
+### Install
 
-> [!IMPORTANT]  
-> Note that as of 2023, the dependencies are very fragile since some of the packages are outdated. This makes installation a bit of a pain.
-
-There are two ways around the painful dependency installation: The first is to try and struggle with npm and all the dependencies to see if you can get them to work. I couldn't, but maybe you can.
-
-The second is to use the alternate JavaScript package manager, `yarn`, since it seems to be much better at managing dependencies and get them to run.
-
-Both sets of instructions are provided here, but instructions on how to install `npm` or `yarn` are not provided. You should choose one or the other, not both!
-
-Note that this only affects the process for building the plugin; You should still follow the instructions below for running the unit tests.
-
-### Building using the Yarn method
-
-Yarn provides a few niceities for running scripts from the directory without installing things globally. All of the dependencies are included in the `package.json` file, including the [Tido `plgToMSS`](https://github.com/tido/plgToMSS) utility and `gulp`, so once you have yarn installed you should be able to run (in the source directory):
-
-```shell
-yarn install
-```
-
-Once this finishes you can test the installation by running:
+For working on Sibmei, you need to have [git](https://git-scm.com/downloads) and [node](https://nodejs.org/en/download) installed. Then, on the command line, run:
 
 ```shell
-yarn run gulp develop
+git clone git@github.com:music-encoding/sibmei.git
+cd sibmei
+npm install
 ```
 
-This will run a file watcher and as you edit the .mss files it should re-build the plugin for you.
-
-> [!WARNING]  
-> As of 2024-07-11 it's impossible to install gulp threw Homebrew. So on newer Macs it is recommended to use npm.
+### Linking the plugin during development
 
-### Building using the NPM method
+You need to create a symbolic link from the Sibelius plugin folder to the build folder in the sibmei directory. The paths are system dependent.
 
-Previous versions of the plugin had you install packages externally, but in this version you can install everything locally within the development directory, including the Tido plgToMSS code (installed from GitHub) and gulp.
+Example for Mac:
 
-```shell
-npm install          // installs the packages listed in the sibmei package.json directory
+```bash
```

Comment:
Why change it to `bash`? If there are no specific bash commands I'd prefer sticking with the more generic `shell`.

Reply:
Agreed. If that's all, can you merge?